doc/distribution.md

```diff
@@ -0,0 +1,104 @@
+# Content Distribution
+
+When an editor publishes content in Writer, it gets distributed across the web. In this doc, we break down the distribution of this content.
+
+![distribute-content-meme](http://files.artsy.net/images/screen-shot-2017-05-25-at-40907-pm.png)
+
+## [Artsy.net](https://www.artsy.net/articles)
+
+We publish all of our content on [Artsy.net](https://www.artsy.net/articles). Each piece of content belongs to a Channel which helps us decide how an article should look.
+
```

Comment:
The first paragraph here seems to need more context -- took a stab:

We publish all editorial content at [Artsy.net/articles](https://www.artsy.net/articles), and articles appear in different contexts throughout Artsy.net. Articles by distinct teams at Artsy, as well as our Gallery and Institutional Partners, are distinguished by their association with a Channel.